src/main/java/net/pms/dlna/DLNAResource.java

```diff
@@ -5016,11 +5017,24 @@ public String getResolutionForKeepAR(int scaleWidth, int scaleHeight) {
 	 * @param file
 	 */
 	private void setMetadataFromFileName(File file) {
+		String absolutePath = file.getAbsolutePath();
+		if (
+			file != null &&
```

Comment:
if `file` is null, then the previous line `file.getAbsolutePath()` would throw a NPE ...

Reply:
Done